### PR TITLE
Move new os issue templates from dotnet/release repo

### PR DESCRIPTION
**Pull request by MichaelSimons**
Related to https://github.com/dotnet/dotnet-docker/issues/5430

### CODE REVIEW

**Review by mthalman — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/new-distro-release.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+1. - [ ] Commit generated changes
+1. - [ ] Create PR
+1. - [ ] Get PR signoff
+1. - [ ] Merge PR to nightly branch
+1. - [ ] Wait for automatically queued CI build to finish on [dotnet-docker-nightly pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=359) (internal MSFT link)
+1. - [ ] Confirm READMEs have been updated in [Docker Hub](https://hub.docker.com/_/microsoft-dotnet)
+
+## Main Branch Tasks
+
+1. - [ ] After the product teams have signed off on the new distro, merge these changes to the main branch as part of the [release process](dotnet-release.md) for the next .NET release
```
Comment (mthalman) [inline .github/ISSUE_TEMPLATE/new-distro-release.md:35]: Reference to dotnet-release.md is obsolete.

File: .github/ISSUE_TEMPLATE/new-windows-release.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+1. - [ ] Run the command to update the image size baseline file: `.\tests\performance\Validate-ImageSize.ps1 -UpdateBaselines`
+1. - [ ] Inspect generated changes for correctness
+1. - [ ] Test the images
+      1. - [ ] Create a local VM of the new Windows version
+      1. - [ ] Clone this repo with the above changes onto the VM
+      1. - [ ] Run `.\build-and-test.ps1 -OS nanoserver-<VERSION>` to build and test your changes for Nano Server
+      1. - [ ] Run `.\build-and-test.ps1 -OS windowsservercore-<VERSION>` to build and test your changes for Windows Server Core
+1. - [ ] Create PR
+1. - [ ] Get PR signoff
+1. - [ ] Merge PR and build/publish as part of the main branch [release process](dotnet-release.md) for the next .NET release
```
Comment (mthalman) [inline .github/ISSUE_TEMPLATE/new-windows-release.md:31]: Reference to dotnet-release.md is obsolete.

File: .github/ISSUE_TEMPLATE/new-distro-release.md (hunk 1 above)
Comment (MichaelSimons) [reply .github/ISSUE_TEMPLATE/new-distro-release.md]: Updated to reference https://github.com/dotnet/release/blob/main/.github/ISSUE_TEMPLATE/dotnet-docker-servicing-release.md.

File: .github/ISSUE_TEMPLATE/new-windows-release.md (hunk 2 above)
Comment (MichaelSimons) [reply .github/ISSUE_TEMPLATE/new-windows-release.md]: Updated to reference https://github.com/dotnet/release/blob/main/.github/ISSUE_TEMPLATE/dotnet-docker-servicing-release.md

**Review by lbussell — APPROVED** [review]

**Pull request merged**